### ISSUE TITLE
Problem with small values

### ISSUE THREAD

**Issue by lbenet** [dpsanders/ValidatedNumerics.jl#136]
I just came across the following:
```julia
julia> using ValidatedNumerics

julia> const epsilon = 1.0e-20
1.0e-20

julia> @interval(epsilon)
Interval(0.0, 0.0)

julia> Interval(epsilon)
Interval(1.0e-20, 1.0e-20)
```

The problem seems to be in `convert` (after expanding the macro):
```julia
julia> convert(Interval{Float64},epsilon)
Interval(0.0, 0.0)
```

**Comment by lbenet** [comment]
The problem is related to the use of `rationalize` in the convert method (line 49 of `src/conversion.jl`:
```julia
julia> rationalize(epsilon)
0//1
```

**Comment by dpsanders** [comment]
Ouch. Do you have a suggestion as to how to solve this?
```@interval("1e-20")``` works.

I do not want to go back to using a string representation for the conversion, since that is very slow.
Maybe we can check if the resulting interval contains the starting float.

**Comment by lbenet** [comment]
No suggestion yet...

**Comment by dpsanders** [comment]
I mean something like
```
II = convert(Interval{Float64}, x)
if !(x ∈ II)
       s = string(x)
       II = convert(Interval{Float64}, string(x))
end
```

**Comment by lbenet** [comment]
Using a small variation of your code, all but one test pass; the failing test is: `x^(1//3) --> x^(1 / 3)`

**Comment by dpsanders** [comment]
I don't understand why that test would fail.

**Comment by dpsanders** [comment]
Do you have a branch with this change?

**Comment by lbenet** [comment]
I've push it only until now, sorry; its called "Iss136".

**Comment by dpsanders** [comment]
Nice catch. The problem is here:
```
julia> convert(Interval{BigFloat}, 1/3)
Interval(3.333333333333332999999999999999999999999999999999999999999999999999999999999968e-01, 3.333333333333333000000000000000000000000000000000000000000000000000000000000012e-01)

julia> convert(Interval{BigFloat}, 1//3)
Interval(3.333333333333333333333333333333333333333333333333333333333333333333333333333305e-01, 3.333333333333333333333333333333333333333333333333333333333333333333333333333348e-01)
```
Clearly the first is wrong.

**Comment by dpsanders** [comment]
Not sure how to find a solution that works for 1/3 and for 1e-20, though. It's possible that for `BigFloat` we'll just have to use the conversion via a string?

**Comment by lbenet** [comment]
Not sure how to solve it. I'm still, playing, but I've been bouncing myself against a wall...

One possibility is leave it as it is, with a note in the documentation and the docstrings.

**Comment by lbenet** [comment]
I didn't notice it before, but there are more failing tests...

**Comment by lbenet** [comment]
Pushed a new commit; not sure why, but locally few tests related to `IntervalBox` are not passing (also in master); probably something related to FixedSizeArrays...

**Comment by lbenet** [comment]
... but not what was problematic before.

**Comment by dpsanders** [comment]
Fixed by #137
